### ISSUE TITLE
Change the Level Display mode

### ISSUE THREAD

**Issue by CCCClear** [Daripher/Auto-Leveling#4]
The current display above the entity has the following problems: it can be blocked by the entity's model, and it can block the player's view when mounted.
Available methods: Compatible display on waila's highlighted display box; Instead of displaying entity level, display area level somewhere on the screen.

**Comment by Namiida0109** [comment]
Since non-hostile creatures are also affected by the level Modpack need to perform complex operations on the blacklist which is hard.
Could U plz add a debug stick,and can add the blaklist by right clicking the entity.
If this is too much work, will it B possible to add collective banning of non-hostile mobs and collective disabling of bosses？

**Comment by Daripher** [comment]
> Since non-hostile creatures are also affected by the level Modpack need to perform complex operations on the blacklist which is hard. Could U plz add a debug stick,and can add the blaklist by right clicking the entity. If this is too much work, will it B possible to add collective banning of non-hostile mobs and collective disabling of bosses？

Please make a separate issue, this is not related to the original post.

**Comment by Daripher** [comment]
> Available methods: Compatible display on waila's highlighted display box;

Can you provide a link to the mod that you are talking about?

**Comment by CCCClear** [comment]
https://www.curseforge.com/minecraft/mc-mods/jade
waila for forge

**Comment by Daripher** [comment]
Added in version 1.11
